app/screens/user_profile/index.js

```diff
@@ -39,6 +39,7 @@ function mapStateToProps(state, ownProps) {
         militaryTime,
         theme: getTheme(state),
         isMyUser: getCurrentUserId(state) === ownProps.userId,
+        remoteClusterInfo: state.entities.remoteCluster.info[user?.remote_id],
```

Comment:
should this be guarded/checked for remoteCluster being `null`?

Reply:
Same for this change. Not a part of this PR. Sorry again